### PR TITLE
Codecov builder

### PR DESCRIPTION
**Pull request by Philmod**
Example: https://github.com/Philmod/google-cloud-build-slack/pull/17

### CODE REVIEW

File: codecov/cloudbuild.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# In this directory, run the following command to build this builder.
+# $ gcloud builds submit . --config=cloudbuild.yaml
+
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '-t', 'gcr.io/$PROJECT_ID/codecov', '.']
+  dir: 'codecov'
```
Comment (nof20) [inline codecov/cloudbuild.yaml]: This `dir` line breaks the build - you're already in the `codecov` directory, so you can just remove it :)

Comment (Philmod) [reply codecov/cloudbuild.yaml]: Removed. The goal was to make it work with the new triggers. See https://github.com/GoogleCloudPlatform/cloud-builders-community/pull/178

**Comment by codrienne** [comment]
Hi @Philmod, I'm going through the backlog of PRs. Looks like I can build the cloudbuild.yaml correctly but the GCB check is failing. PTAL, or LMK if you don't plan on working on this soon and I'll close the PR (can always re-open or file a new one later).

**Comment by codrienne** [comment]
Closing old PRs from before Q3. Please re-open or file a new PR at will.

**Comment by Philmod** [comment]
Sorry @codrienne I missed your previous message. I don't have the permission anymore to see this build's logs anyway.

**Pull request closed without merging**